### DISCUSSION TITLE
GPU acceleration

### DISCUSSION THREAD

**Discussion by LGDi** [e0404/matRad#655] [Ideas]
Dear developers,
Is there any interface related to GPU acceleration or GPU Monte Carlo simulation in matRad?

**Comment by wahln** [comment]
There's not yet a GPU-accelerated MC interface yet. Do you have a specific code in mind which you would like to see interfaced?

**Comment by LGDi** [comment]
I'm sorry.I don't have a complete program and I also want to achieve GPU-accelerated MC.

**Comment by wahln** [comment]
We will not program our own GPU-MC code and also don't use one at the moment. So feel free to look for available GPU-accelerated MC codes. If you find some you would be interested to use, we might be able to help with implementing the interface.
